mix.exs

```diff
@@ -60,9 +60,12 @@ defmodule SpandexPhoenix.MixProject do
     [
       {:excoveralls, "~> 0.10", only: :test},
       {:ex_doc, ">= 0.0.0", only: :dev},
-      {:git_ops, "~> 2.0.0", only: :dev},
+      {:git_ops, "~> 2.0", only: :dev},
       {:inch_ex, "~> 2.0", only: [:dev, :test]},
+      {:jason, "~> 1.0", only: [:dev, :test]},
+      {:optimal, "~> 0.3"},
```

Comment:
Note: I added this so that we could theoretically retire the previous version and have people use this one safely along with an updated spandex that doesn't include it anymore.